core/src/main/java/io/undertow/server/handlers/Cookie.java

```diff
@@ -75,6 +85,7 @@ default boolean isSameSite() {
         return false;
     }
 
+    @Deprecated(forRemoval = true)
```

Comment:
Any one understand what could be the use case to have value and set this flag to false?